db/vita_partners.yml

```diff
@@ -484,3 +484,24 @@ vita_partners:
   source_parameters:
     - uwsuncoast
   logo_path: 'partner-logos/UWSC.png'
+- name: Temple University
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360012561073'
+  display_name: Temple University
+  source_parameters:
+  - tuvs
+  logo_path: 'partner-logos/temple.jpg'
+- name: Goodwill of the Olympics and Rainier Region
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360012588394'
+  display_name: Goodwill of the Olympics and Rainier Region
+  source_parameters:
+  - orrr
```

Comment:
@GAV07 does `orrr` have the correct number of `r`s? it looks like the rest hints at just `orr`

Reply:
Yes it is correct although I am not sure why they would have added an extra r instead of just two.

Reply:
https://www.getyourrefund.org/?s=orrr (from [airtable](https://airtable.com/tbl5epnGDYQf8OAYZ/viwgXpuhMweqBKPIP/reck0uQ1GZjhbstX1?blocks=bipsOP1FkYFvEXxjd))